cmd/lncli/commands.go

```diff
@@ -2170,11 +2171,11 @@ func updateChannelPolicy(ctx *cli.Context) error {
 		}
 	}
 
-	if feeRate != 0 {
-		req.FeeRate = feeRate
-	} else if feeRatePpm != 0 {
-		req.FeeRatePpm = uint32(feeRatePpm)
-	}
+	// set both feerate values. Since we have
```

Comment:
Please capitalize all sentences and use the full available 80 characters of line length (while setting the editor to use 8 spaces per tab character).